Src/DependencyCollector/Shared/Implementation/SqlClientDiagnostics/SqlClientDiagnosticSourceListener.cs

```diff
@@ -301,7 +306,7 @@ public void Dispose()
                                 Type = RemoteDependencyConstants.SQL,
                                 Target = string.Join(" | ", connection.DataSource, connection.Database),
                                 Data = operation,
-                                Success = true
+                                Success = true,
```

Comment:
intentional?

Reply:
Not intentional, but it is a coding style from my projects.  I'll remove the trailing comma when we hear back regarding the feature flag.